src/App/Header/Search.svelte

```diff
@@ -23,7 +23,12 @@
   router.historyStore.subscribe(() => (input = ""));
 
   const collapsedWidth = 13;
-  const expandedWidth = 20;
+  let searchBarWidth = collapsedWidth;
+
+  function expandSearchBar() {
+    const isMobile = window.matchMedia("(max-width: 720px)").matches;
```

Comment:
Is there some reason we need this to be a js function doing css?
Couldn't we toggle between two CSS classes with different widths `on:focus` on the div wrapping the `TextInput`.
Something like 
```css
.expandedWidth {
 width: 20rem;
}
.collapsed {
  width: 13rem;
}
```
And the case when on mobile should be possible to be done with CSS media queries

Reply:
Theoretically yes, but I didn't want for the search bar to contract when it looses focus when a search is in progress. This would also happen when a search fails.

Reply:
Yeah, good idea. 90b04ebf2484fbcadcbd29c42da4651edd56b329